test/fixtures/user/user.js

```diff
@@ -144,6 +144,9 @@ module.exports = () => {
         app_owner: true,
         archived: true,
       },
+      colors: {
+        color_id: 2,
+      },
```

Comment:
not sure ATM, we can make it just color_id, but if we are going to add more colors in the future then this would be better,  should i change the property?